mods/gen4/formats-data.js

```diff
@@ -467,7 +467,7 @@ exports.BattleFormatsData = {
 		tier: "NU",
 	},
 	hitmontop: {
-		randomBattleMoves: ["fakeout", "suckerpunch", "machpunch", "bulkup", "rapidspin", "closecombat", "stoneedge", "pursuit", "bulletpunch", "foresight"],
```

Comment:
It's really weak, even with Technician. Randbat's balanced EV spreads make weak priority moves a lot less effective. LO Technician Bullet Punch only does 30-35% to Gengar, for example, and Hitmontop won't always roll LO or Technician. Any of the other moves in its movepool will almost always be better.